src/chain.cpp

```diff
@@ -80,12 +80,13 @@ int static inline GetSkipHeight(int height) {
     return (height & 1) ? InvertLowestOne(InvertLowestOne(height - 1)) + 1 : InvertLowestOne(height);
 }
 
-CBlockIndex* CBlockIndex::GetAncestor(int height)
+const CBlockIndex* CBlockIndex::GetAncestor(int height) const
 {
-    if (height > nHeight || height < 0)
+    if (height > nHeight || height < 0) {
```

Comment:
Nit, kind of unrelated since this is not touched.

Reply:
True, but from what I've seen so far it's much easier to improve style is in pull requests which actually also change functionality. Usually I would do that in a separate commit, but this is such a small change that I thought it made sense to just add it to the single commit.